### ISSUE TITLE
Interactive element does not meet minimum size nor spacing

### ISSUE THREAD

**Issue by alomarramaa** [samvera/hyku#3091]
http://vpat.hykudemo.org
Version 7.1

### Descriptive Summary
If a link/button is smaller than 24 x 24 pixels, it will be difficult for users to tap. All links/buttons should be minimum 24 x 24 pixels.

### Expected Behavior
Ensure all buttons/links are at least 24 x 24 pixels in the code.

### Steps to Reproduce Issue
1. Go to vpat.hykudemo.org
2. Scan website using SiteImprove browser extension

### Occurrences of Issue

Cultural Home Repository (2 occurances)

<img width="577" height="310" alt="Image" src="https://github.com/user-attachments/assets/8a061dfb-b4ed-4cb3-a5d1-a566cf346548" />

Institutional Repository Home Page

<img width="566" height="239" alt="Image" src="https://github.com/user-attachments/assets/0eab4133-0739-4b9b-a78e-c158e5400571" />

Neutral Repository Home Page

<img width="562" height="310" alt="Image" src="https://github.com/user-attachments/assets/855c33c9-9896-46a4-a7e3-0e86b2a37060" />

Cultural Show Page at https://vpat.hykudemo.org/concern/etds/9589c183-af3e-440f-8d1d-4e0c96a35a67

<img width="570" height="359" alt="Image" src="https://github.com/user-attachments/assets/d773812f-3372-41ff-bf87-b2ea0faaf8de" />
